tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
+envlist = py27,py34,py35,py36
+
+[testenv]
+deps=pytest
+commands=py.test
+
+[testenv:py27]
+deps = -rrequirements-py2.txt
```

Comment:
looks like a double r in rrequirements-py2.txt

Reply:
It's a flag `-r` plus `requirements-py2.txt`
http://tox.readthedocs.io/en/latest/example/basic.html#depending-on-requirements-txt-or-defining-constraints